CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 6.7.0
+__30.05.20205__
+
+- See the changelog for 6.7.0-dev.1 for details.
```

Comment:
What about #747, #748, #749 and #750?

Reply:
I think we should avoid avoid referencing other versions like that all together. Just copy what ever was release in dev version and if needed add new stuff